core/src/main/scala/cats/Alternative.scala

```diff
@@ -83,6 +83,10 @@ import scala.annotation.implicitNotFound
   def guard(condition: Boolean): F[Unit] =
     if (condition) unit else empty
 
+  override def zero: F[cats.data.INothing] = empty
+
+  override def choice[A, B](fa: F[A], fb: F[B]): F[Either[A, B]] = sum(fa, fb)(this)
```

Comment:
A nitpick, even if equivalent here, for consistency with the rest of the file, how about
```suggestion
  override def choice[A, B](fa: F[A], fb: F[B]): F[Either[A, B]] = sum(fa, fb)(self)
```